### ISSUE TITLE
Model Imports are failing in production

### ISSUE THREAD

**Issue by CraigJZ** [rubyforgood/abalone#793]
The CSV Upload tool for Animals, Enclosures and Cohorts is working in development but failing in production with the following error:

`Error performing ImportAnimalsJob (Job ID: 123456789) from DelayedJob(default) in 5.79ms: ActiveStorage::FileNotFoundError (ActiveStorage::FileNotFoundError)`

We will need to determine what is happening in production to cause this failure and refactor the code to resolve.

This issue is also a good place to discuss the placement of these.  Currently they reside on index of each item. This may be confusing from a user experience perspective since we also have the separate Uploads UI. Should these uploads happen through that interface? If a fix for this issue overlaps into a potential refactoring of the UI, we can create additional issues or update this one.

**Comment by adub65** [comment]
@CraigJZ , I uploaded a few CSV in the staging environment and I dont get an error. However, the status says "Pending". Is there a certain time frame that the status should update? Or does the constant pending status mean something is wrong?

**Comment by CraigJZ** [comment]
@adub65 Yes, it stays stuck on a pending status but in the logs we see an error similar to:
```
Error performing ImportAnimalsJob (Job ID: 123456789) from DelayedJob(default) in 5.79ms: ActiveStorage::FileNotFoundError (ActiveStorage::FileNotFoundError)
```
